src/Api/SecretsManager/Controllers/CountsController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Api.SecretsManager.Models.Response;
+using Bit.Core.Context;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.SecretsManager.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.SecretsManager.Controllers;
+
+[Authorize("secrets")]
+[Route("organizations/{organizationId}")]
+public class CountsController : Controller
+{
+    private readonly ICurrentContext _currentContext;
+    private readonly IUserService _userService;
+    private readonly IProjectRepository _projectRepository;
+    private readonly ISecretRepository _secretRepository;
+    private readonly IServiceAccountRepository _serviceAccountRepository;
+
+    public CountsController(
+        ICurrentContext currentContext,
+        IUserService userService,
+        IProjectRepository projectRepository,
+        ISecretRepository secretRepository,
+        IServiceAccountRepository serviceAccountRepository)
+    {
+        _currentContext = currentContext;
+        _userService = userService;
+        _projectRepository = projectRepository;
+        _secretRepository = secretRepository;
+        _serviceAccountRepository = serviceAccountRepository;
+    }
+
+    [HttpGet("counts")]
+    public async Task<OrganizationCountsResponseModel> GetByOrganizationAsync([FromRoute] Guid organizationId)
+    {
+        if (!_currentContext.AccessSecretsManager(organizationId))
+        {
+            throw new NotFoundException();
+        }
+
+        var userId = _userService.GetProperUserId(User).Value;
+        var orgAdmin = await _currentContext.OrganizationAdmin(organizationId);
+        var accessType = AccessClientHelper.ToAccessClient(_currentContext.ClientType, orgAdmin);
+
```

Comment:
There is a query to simplify this.

```suggestion
   var (accessType, userId) = await _accessClientQuery.GetAccessClientAsync(User, organizationId);
```

Reply:
Out of scope for this PR, but it would be good to look into making a class `AccessClient` that we can pass around instead of having to pass the `userID` and `accessType` everywhere.

Reply:
Fixed in https://github.com/bitwarden/server/pull/4200/commits/41b166c6059a33ccd52066d0b524f387254ffa0d